aten/src/ATen/native/TensorAdvancedIndexing.cpp

```diff
@@ -68,7 +69,99 @@
 #include <numeric>
 #include <vector>
 
-namespace at { namespace native {
+namespace at {
+namespace meta {
+
+native::SCATTER_GATHER_OP get_operator_enum(const c10::string_view reduce) {
```

Comment:
Really good use case for #55526 since string comparison is nontrivially expensive

Reply:
(no action needed from you)